src/MergeContext.js

```diff
@@ -938,9 +941,61 @@ class MergeFinalizer extends MergeContext {
         }
     }
 
+    // Start processing
```

Comment:
Since "processing" can be interpreted in many ways (metadata updates, staging, merging, staging+merging), I suggest removing these "Start processing" and "Finish processing" labels. Also, grouping methods together like this rarely works long-term. Sooner or later, some of the "start processing" methods will be added elsewhere.